packages/runtime/src/state/makeswift-api-client.ts

```diff
@@ -129,5 +132,20 @@ export type Dispatch = ThunkDispatch<State, unknown, Action>
 export type Store = ReduxStore<State, Action> & { dispatch: Dispatch }
 
 export function configureStore({ serializedState }: { serializedState?: SerializedState }): Store {
-  return createStore(reducer, APIResources.getInitialState(serializedState), applyMiddleware(thunk))
+  const composeEnhancers = composeWithDevToolsDevelopmentOnly({
+    name: `API client store (${new Date().toISOString()})`,
+    serialize: true,
+    stateSanitizer: (state: any) => serializeState(state),
+    actionsBlacklist: [
+      ActionTypes.BUILDER_POINTER_MOVE,
+      ActionTypes.HANDLE_POINTER_MOVE,
+      ActionTypes.ELEMENT_FROM_POINT_CHANGE,
+    ],
```

Comment:
Don't log actions that fire on every mouse move.

Reply:
If these actions become important for debugging, might it be confusing if they're missing? Is there perhaps a way to collapse these in DevTools if they occur all together?

Reply:
> Is there perhaps a way to collapse these in DevTools if they occur all together?

AFAIK, there isn't. 

I think it's more or less expected that "spamming" actions are by default excluded from log output, and that you need to change something in code (or the console) that get the "verbose" output.